packages/build/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Lines starting with '#' are comments.
```

Comment:
Ditto, CODEOWNERS file does not make sense inside a package. Please move this entry to top-level CODEOWNERS file at monorepo's root level and use a path specifier to limit the codeowners to this package only.